inspec.yml

```diff
@@ -1,9 +1,9 @@
-name: {{plan_name}}
-title: Habitat Core Plan {{plan_name}}
-maintainer: humans@habitat.sh
-summary: InSpec controls for testing Habitat Core Plan {{plan_name}}
-copyright: humans@habitat.sh
-copyright_email: humans@habitat.sh
+name: diffutils
+title: Habitat Core Plan diffutils
+maintainer: "The Habitat Maintainers <humans@habitat.sh>"
+summary: InSpec controls for testing Habitat Core Plan diffutils
+copyright: 2019, Chef Software, Inc.
```

Comment:
Can this be updated to match the latest skeleton inspec.yml?

Reply:
Done.